shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -325,7 +325,7 @@ private MotionEvent toMotionEvent(float density, PlatformViewsChannel.PlatformVi
       return MotionEvent.obtain(
           trackedEvent.getDownTime(),
           trackedEvent.getEventTime(),
-          trackedEvent.getAction(),
+          touch.action, // TODO (kaushikiska): https://github.com/flutter/flutter/issues/61169
```

Comment:
Offline discussion: the recommended fix is to leave the virtual display code path unchanged, and have a new `toMotionEvent` method for hybrid composition, which uses `trackedEvent.getAction()`.

Also, we will need unit tests since the e2e test in https://github.com/flutter/flutter/tree/master/dev/integration_tests/android_views failed at capturing this issue.  I will investigate this issue.